index.js

```diff
@@ -10,9 +10,7 @@ Object.defineProperty(exports, 'commentRegex', {
 
 Object.defineProperty(exports, 'mapFileCommentRegex', {
   get: function getMapFileCommentRegex () {
-    //Example (Extra space between slashes added to solve Safari bug. Exclude space in production):
-    //     / /# sourceMappingURL=foo.js.map           /*# sourceMappingURL=foo.js.map */
-    return /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
+    return /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"`]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
   }
```

Comment:
Could we please keep the comments that were there?

Reply:
How about something more like:
```js
// Matches sourceMappingURL in either // or /* comment styles.
```

The current comment interferes with global packages where two installations of `convert-source-map` end up parsing each other, causing an error to be thrown.